app/controllers/discover_controller.rb

```diff
@@ -0,0 +1,3 @@
+class DiscoverController < ApplicationController
+  def show; end
```

Comment:
Wait, for real?!  This is what rubocop actually wanted?!  #shocked!

Reply:
I know! It said something like "empty actions should be one line"